routes/story.route.server.js

```diff
@@ -80,7 +81,79 @@ module.exports = app => {
                 .then(story => res.json(story))
                 .catch((error) => res.status(500).send({error}));
     };
-  
+
+    // there is no check for userId being valid here. The expectation is that only validated users would be able to
+    // navigate to comment page
+    addComment = (req,res) => {
```

Comment:
Might be a good idea to split comments related code to a new route file?

Reply:
yes, can be done, will add it as backlog for the moment